### DISCUSSION TITLE
Status or StatusId

### DISCUSSION THREAD

**Discussion by AbbeyDesign** [solspace/craft-freeform#345] [Questions & Answers] [answered]
How do I, on a front-end form, change the status of a submission. On a select field, I have tried name="status" and name="statusId" with option values being either the handle or the id.

In the cp, Freeform uses the statusId in a hidden field but I can't get it to work on the front end.

**Comment by kjmartens** [comment, marked as answer]
Hi @AbbeyDesign,

Currently, the only way to change statuses is via the `status` parameter. There is no input field option at this time. It's on our list, however, to come up with a solution for this kind of flow. 🙂

**Comment by AbbeyDesign** [comment]
Ok, thanks!
